ketchup/main.py

```diff
@@ -1,43 +1,62 @@
+from dateutil import parser
 import datetime as dt
 from ketchup.luffy import LUFFY
-import ketchup.day_counter as dc
+import ketchup.watch_schedule_builder as dc
 import ketchup.fun_fact as fun_fact
 
 
 def run():
-    date_format = "%Y,%m,%d"
-
-    episodes = input(
-        "Hello! How many episodes is the show you are interested in watching? "
-    )
+    episodes = int(input(
+        "Hello! How many episodes are in the show you are interested in watching? "
+    ))
 
     end_date = input(
         "What date would you like to have completed this series?(yyyy,mm,dd) "
     )
-    end_date = dt.datetime.strptime(end_date, date_format)
+    end_date = parser.parse(end_date)
 
     answer_today = input("Will you start watching today? [Y/n] ") or "y"
     if answer_today.lower() == "y":
-        start_date = None
+        start_date = dt.datetime.today()
     else:
```

Comment:
If the user answers 'n' and then provides an eligible date, the math is seeming to be incorrect. 

<img width="1207" alt="Screenshot 2023-08-13 at 17 24 06" src="https://github.com/chellygel/AnimeKetchup/assets/121133431/ce7285fd-4c3b-44ec-80ad-2ccab48dccc5">

Today is not one day from the 2024/12/08, instead there are 483 days until then.

Reply:
i'll leave this to a future iteration to fix but i did change the wording to say

`"Wow, that's only {watch_schedule['days']} days between today and then !!!"` until then.